src/sass/modules/_m-form-elements.scss

```diff
@@ -55,7 +55,7 @@ button.form-button-disabled {
     display: block;
     float: left;
     margin-left: -1.7em;
-    margin-top: 0.14em;
+    margin-top: 0.25em;
```

Comment:
Seems odd that the labels were too low, but increasing the `margin-top` fixes it. What's up with that? I feel like I'm probably missing something.